src/main/java/com/ase/restservice/model/Asset.java

```diff
@@ -0,0 +1,30 @@
+package com.ase.restservice.model;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name="asset")
+public class Asset {
+    @EmbeddedId
+    private AssetId assetId;
```

Comment:
Why do we need a separate class for AssetId? Doesn't having account_id and stock_id fields here represent the database better?